pkg/network/deviceinfo/sriov.go

```diff
@@ -57,45 +55,30 @@ func CreateNetworkPCIAnnotationValue(networks []v1.Network, interfaces []v1.Inte
 
 func mapNetworkNameToPCIAddress(networks []v1.Network, interfaces []v1.Interface,
 	networkStatusAnnotationValue string) (map[string]string, error) {
-	multusInterfaceNameToNetworkStatusMap, err := mapMultusInterfaceNameToNetworkStatus(networkStatusAnnotationValue)
+	networkNameToDeviceInfo, err := mapNetworkNameToDeviceInfo(
+		networks,
+		networkStatusAnnotationValue,
+		vmispec.FilterSRIOVInterfaces(interfaces),
```

Comment:
Please note `vmispec.FilterSRIOVInterfaces` is also called at line 68, could you save its result and reuse?
Or, if its possible,  use `networkNameToDeviceInfo` at line 68 instead.